frontend/src/actions/send/abandon_action.ts

```diff
@@ -12,7 +12,6 @@ class AbandonMessage implements Message {
 function AbandonAction(repository: ConnectionRepository) {
     const m = new AbandonMessage()
     repository.sendWebSocketMessage(m)
-    //repository.closeWebSocketConnection()
```

Comment:
If a user sends an AbandonAction, does he also receive the Abandon response from backend? In case he doesn't, this send action should close the Websocket.

In general, I consider the Websocket must be closed in this action no matter the backend response.  What do you think? Are any drawbacks in doing it like I mention?